client/src/main/java/io/pravega/keycloak/client/KeycloakAuthzClient.java

```diff
@@ -139,14 +139,10 @@ public String getRPT() {
                     LOG.warn("Retryable HttpResponseException with HTTP code:  {}", statusCode);
                     return true;
                 }
-            } else if (rootCause instanceof ConnectException || rootCause instanceof UnknownHostException) {
-                LOG.warn("Retryable connection exception", rootCause);
-                return true;
-            } else {
-                    // random unexpected Exceptions, don't retry, these should be debugged.
-                    LOG.error("Other non retryable exception", rootCause);
-                    return false;
-                }
+            }
+            LOG.warn("Retryable exception: ", e);
+            LOG.warn("Retryable exception root cause: ", rootCause);
+            return true;
```

Comment:
Deletion of lines 142-149, and default of returning true at the end of the predicate is the core of this PR.